client-v2/src/components/FrontsEdit/CollectionComponents/Collection.tsx

```diff
@@ -192,15 +194,20 @@ class Collection extends React.Component<CollectionProps> {
                 <Button
```

Comment:
Could we have the same behaviour for Discard?

Reply:
You mean so that the spinner runs if you are discarding a set of changes? 
Can do - do you think the delay there is long enough to merit the loading spinner?

Reply:
do you still want this..? Can add

Reply:
Would we ever have a delay when discarding?

Reply:
This would be awesome to have. Discard involves a network call, so this button is latent for network + server time 👍

Reply:
(If it's too much it can always be a separate PR)